CONTRIBUTING.md

```diff
@@ -24,58 +24,48 @@ Thank you so much for showing interest in contributing to TagStudio! Here are a
 
 ### Prerequisites
 
-- [Python](https://www.python.org/downloads/) 3.12
-- [Ruff](https://github.com/astral-sh/ruff) (Included in `requirements-dev.txt`)
-- [Mypy](https://github.com/python/mypy) (Included in `requirements-dev.txt`)
-- [PyTest](https://docs.pytest.org) (Included in `requirements-dev.txt`)
+- [Python](https://www.python.org/downloads/) 3.11 or 3.12
+- [Poetry](https://python-poetry.org/docs/)
 
-### Creating a Python Virtual Environment
+### Installing dependencies 
 
-If you wish to launch the source version of TagStudio outside of your IDE:
+After installing poetry and python you can install the dependencies with the following command:
```

Comment:
Im in favor of sharing one way to install a tool if there is consensus to use it. I use poetry daily, there are many things I don't like about it, but it makes the interface we use the same for all developers, no matter what platform you are on.

Currently the recommended way to install poetry is by using `pipx` this is a tool that is recommended for python devs to have, but it is yet another install process to document. 

Im happy write the changes needed if that is something you would like me to do.

Reply:
It is not clear to me what changes you are actually suggesting here. @eivl

Reply:
`pipx install poetry` is the command to install poetry. the docs has multiple other ways to install it, but I'm in favor of front loading the information instead of back loading it with more links to click on.